packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -65,22 +79,59 @@ export default class DropdownCore extends React.Component<DropdownCoreProps> {
     }
 
     componentDidMount() {
+        this.updateEventListeners();
+    }
+
+    componentDidUpdate(prevProps: DropdownCoreProps) {
+        if (prevProps.open !== this.props.open) {
```

Comment:
You have a few props in this method, might be good to do: `const {open, alignment} = this.props;` to save a few characters.

Reply:
Got rid of the other props here ;D, so leaving this.